autogen/src/main/java/org/apache/hadoop/fs/dfsioe/TestDFSIO.java

```diff
@@ -70,7 +72,7 @@
   private static final String BASE_FILE_NAME = "test_io_";
   private static final String DEFAULT_RES_FILE_NAME = "TestDFSIO_results.log";
   
-  private static final Log LOG = FileInputFormat.LOG;
+  private static final Logger LOG = LoggerFactory.getLogger(TestDFSIO.class);;
```

Comment:
Why changing logging class?

Reply:
@gczsjdy  FileInputFormat.LOG in hadoop 2 is different from hadoop 3.   Hadoop 3 used org.slf4j.Logger however hadoop 2 used org.apache.commons.logging.log, so changing logging class for compatibility.

Reply:
Great!